packages/patternfly-4/react-core/src/components/Accordion/Accordion.tsx

```diff
@@ -24,6 +24,5 @@ export const Accordion: React.FunctionComponent<AccordionProps> = ({
 }: AccordionProps) => (
   <dl className={css(styles.accordion, className)} aria-label={ariaLabel} {...props}>
     <AccordionContext.Provider value={headingLevel}>{children}</AccordionContext.Provider>
-    {children}
```

Comment:
Great catch, this should have never made it in...

Reply:
Thank you!